### PR TITLE
docs: add contributing guide

### PR DESCRIPTION
**Pull request by krunalcodes**
## What does this PR do?

Adds contribution guide to docs.cal.com

Fixes # (issue)
#1944 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?

Open docs site and go to `/contributing`. Contribution guide should be visible.

### CODE REVIEW

File: apps/docs/pages/contributing.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+- Before jumping into a PR be sure to search [existing PRs](https://github.com/calcom/cal.com/pulls) or [issues](https://github.com/calcom/cal.com/issues) for an open or closed item that relates to your submission.
+
+## Developing
+
+The development branch is `main`. This is the branch that all pull
+requests should be made against. The changes on the `main`
+branch are tagged into a release biweekly.
```
Comment (baileypumfleet) [inline apps/docs/pages/contributing.mdx:15]:
```suggestion
branch are tagged into a release monthly.
```

File: apps/docs/pages/contributing.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+```bash
+yarn build
+```
+
+Please be sure that you can make a full production build before pushing code.
+
+## Testing
+
+More info on how to add new tests comming soon.
```
Comment (baileypumfleet) [inline apps/docs/pages/contributing.mdx:58]:
```suggestion
More info on how to add new tests coming soon.
```

**Review by pumfleet — CHANGES_REQUESTED** [review]
This is really great! Left a couple of minor changes here, but otherwise, this is good to go :) Thank you

**Review by pumfleet — APPROVED** [review]

**Comment by imkrunal** [comment]
@baileypumfleet Those changes need to be done in main `CONTRIBUTING.md` as well right?

**Comment by baileypumfleet** [comment]
> @baileypumfleet Those changes need to be done in main `CONTRIBUTING.md` as well right?

Yes good point

**Comment by imkrunal** [comment]
@baileypumfleet #2036

**Pull request merged**